services/motion/pbhelpers.go

```diff
@@ -345,3 +325,17 @@ func (r MoveOnMapReq) toProtoNew(name string) (*pb.MoveOnMapNewRequest, error) {
 
 	return req, nil
 }
+
+// incompletePlan is a plan without any Trajectory information and will return an empty Trajectory if it is requested.
+// It is necessary since the proto definition for a plan does not contain this information. Use with caution.
+type incompletePlan struct {
```

Comment:
This might be better as a `motionplan.SimplePlan`, which can be initialized, wrapping `path` and `trajectory` if they exist and can be initialized with `Trajectory{}` if not. The same struct can be used to resolve the above discussions regarding only accepting `*rrtPlan`

Reply:
I like this idea because its a good idea.  But I also like it because I used to like to listen to Simple Plan so maybe I'm biased

Reply:
Ok I've added it now,  let me know if that works better for you